test/unit/authInfoTest.ts

```diff
@@ -1452,6 +1452,10 @@ describe('AuthInfo', () => {
   describe('audienceUrl', () => {
     const sfdxAudienceUrlSetting = process.env.SFDX_AUDIENCE_URL;
 
+    beforeEach(() => {
+      process.env.SFDX_AUDIENCE_URL = '';
```

Comment:
I mock this out since I have `process.env.SFDX_AUDIENCE_URL` set for internal env. Tests should not depend on the actual `process.env.SFDX_AUDIENCE_URL `